packages/clerk-js/src/utils/image.ts

```diff
@@ -0,0 +1,18 @@
+import { isomorphicAtob } from '@clerk/shared';
+import type { EncodedImageData } from '@clerk/types';
+
+export const isDefaultImage = (url: string) => {
+  // Remove this check when renaming experimental_imageUrl to imageUrl
```

Comment:
Should this be a `TODO:` not sure if that would help to discover easier at a later point

Reply:
Done